learn/guides/configure-ballerina-programs/provide-values-to-configurable-variables.md

```diff
@@ -30,6 +30,7 @@ given through multiple ways when retrieving configurable values:
     specifying multiple configuration files using this environment variable with the OS-specific separator. The file
     precedence order will be as specified in the environment variable. If an environment variable is not specified, the file
     will be located in the current working directory with the file name `Config.toml` by default.
+>**Note:** Configuration values for testing can be provided in a file named `Config.toml` located in the `tests` directory.
```

Comment:
Should this be a another point intead of note?

Configure Ballerina Tests: Configuration values for testing can be provided in a file named `Config.toml` located in the `tests` directory.

Reply:
We already have more information about this in the `testing` learn page. 
https://ballerina.io/learn/test-ballerina-code/configure-tests/#define-test-specific-configurations

Shall I add a link to that page instead of explaining here in detail?